pyrdp/layer/tcp.py

```diff
@@ -57,7 +57,14 @@ def connectionMade(self):
         When the TCP handshake is completed, notify the observer.
         """
         self.connectedEvent.set()
-        self.observer.onConnection()
+
+        try:
+            # Connection to client
+            ip = self.transport.client[0]
+            self.observer.onConnection(ip)
+        except AttributeError:
```

Comment:
There must be a cleaner way to do that 🤔

Reply:
I haven't seen anything defined at this moment (or before) that allowed me to be 100% sure that the current connection is a client. Still, I'll have another look.

Reply:
I overlooked a small detail, much simpler and cleaner now. Thanks! :slightly_smiling_face: